github/functions-big-merge.sh

```diff
@@ -1,6 +1,13 @@
 # This file contains overwrite for the functions in functions.sh
 set +x
 
+# Prefixes the old version of the given function with "_" to make it callable
+# from the overwriting function.
+function overwrite_function() {
+  local function_name=$1
+  eval "`declare -f $function_name | sed '1s/.*/_&/'`"
```

Comment:
```suggestion
  eval "_`declare -f $function_name`"
```
Would this work? I'm not really clear on what that `sed` call does.